lib/src/fnc/util/math/variance.rs

```diff
@@ -1,11 +1,17 @@
 use crate::sql::number::Number;
 
 pub trait Variance {
+	/// Population Variance of Data
```

Comment:
Maybe worth looking here: https://github.com/surrealdb/surrealdb/blob/golang/util/math/variance.go

We used to have different variance functions...

Reply:
Nothing wrong - just FYI!